lmdeploy/lite/apis/calibrate.py

```diff
@@ -118,7 +119,11 @@ def calibrate(model: str,
               calib_samples: int = 128,
               calib_seqlen: int = 2048,
               work_dir: str = './work_dir',
-              device: str = 'cuda') -> None:
+              device: str = 'cuda',
+              w_bits: int = 4,
```

Comment:
I don't think we should expose `w_bits` since we only support 4bit weight-only quantization now.